kmm/read.py

```diff
@@ -34,27 +36,28 @@ def read(path: Path, header_path: Path = None):
 
     if header_path is not None:
         df = camera_positions(df, header_path)
```

Comment:
Should this be decided by an argument?

Reply:
`header_path` can no longer be None, this is left over :) Ty for noticing!